### ISSUE TITLE
Geary > Dark mode > Text in replies is barely visible

### ISSUE THREAD

**Issue by WatchMkr** [pop-os/gtk-theme#344]
This appears to be a recent change. Text was visible in replies before updating the theme today.

**Comment by isantop** [comment]
This also happens in Adwaita, so it will require a lot of digging to fix.

**Comment by WatchMkr** [comment]
Running Pop 19.04, this version of the theme doesn't have the bug:

```pop-gtk-theme:
  Installed: 4.1.4~1560290633~19.04~f75e86a
  Candidate: 4.1.4~1560290633~19.04~f75e86a
  Version table:
 *** 4.1.4~1560290633~19.04~f75e86a 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main i386 Packages
        100 /var/lib/dpkg/status
```

Not sure if it's present if I upgraded to 19.10 so it could be a change to Geary. The main difference is that the text window in Geary is white vs black in dark mode.

**Comment by WatchMkr** [comment]
@isantop after updates today this bug is gone. Please confirm that it's the same for you.

**Comment by isantop** [comment]
![Screenshot from 2019-10-14 09-55-00](https://user-images.githubusercontent.com/5883565/66765239-b8720380-ee68-11e9-8df4-945f51416ddd.png)
I'm still getting this issue

**Comment by isantop** [comment]
Upstream bug report: https://gitlab.gnome.org/GNOME/geary/issues/573

This appears to have been fixed upstream and is present in Geary 3.35 (Pop uses 3.32)

**Comment by isantop** [comment]
This is now fixed in staging thanks to pop-os/geary

I'm thus closing this issue.
